gomatrixserverlib/pipeline.yml

```diff
@@ -0,0 +1,21 @@
+steps:
+  - command:
+      - "env"
```

Comment:
is this actually useful, or just noise?

Reply:
It does include the exact Go minor version of the image, but I suppose that can be replaced with `go version` if that's preferable.